data/json/items/ammo/454.json

```diff
@@ -0,0 +1,25 @@
+[
```

Comment:
Just FYI, making these so small is probably going to have bad effects on load times, deserialization works best when it can grab a lot of data at a time.  I have an idea to concatenate all the json together and compress it to improve load times, that might mitigate/eliminate the issue.